apps/web/src/components/mf/ChatInput/index-bak.tsx

```
@@ -0,0 +1,265 @@
+          </button>
+        ) : loading ? (
+          <button
+            className={clsx(styles.sendBtn, styles.loading)}
+            onClick={(e) => {
+              if (loading) {
+                onStop && onStop()
+              }
+            }}>
+            <span className={styles.stopSquare}></span>
+          </button>
+        ) : (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)